eta/core/image.py

```diff
@@ -701,6 +701,17 @@ def resize(img, width=None, height=None, *args, **kwargs):
     return cv2.resize(img, (width, height), *args, **kwargs)
 
 
+def central_crop(img, h, w):
+    '''Crops the central part of an image and outputs an image of shape (h, w)
```

Comment:
For top-level public functions, we prefer to use a full docstring in Google python style:
```py
'''Description here.

Args:
    img: xxx
    (h, w): yyy

Returns:
    description here
'''
```

Reply:
Oh, didn't note this. I just followed the docstring format of the function above this one `resize`.